site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -237,27 +247,36 @@ In this example, we are using message address `t/DEFAULT_TENANT/4711` which cont
 
 ## Publishing Events
 
-The adapter supports *AT LEAST ONCE* delivery of *Event* messages only. A client therefore MUST set the *settled* property to `false` and the *rcv-settle-mode* property to `first` in all *transfer* frame(s) it uses for uploading events. All other combinations are not supported by the adapter and result in the message being rejected.
+The adapter supports *AT LEAST ONCE* delivery of *Event* messages only. A client therefore MUST set the *settled*
+property to `false` and the *rcv-settle-mode* property to `first` in all *transfer* frame(s) it uses for uploading events.
+All other combinations are not supported by the adapter and result in the message being rejected.
 
 ## Publish an Event (authenticated Device)
 
-* Message Address: `event` or `e`
-* Settlement Mode: `unsettled` (AT LEAST ONCE)
 * Authentication: SASL PLAIN or SASL EXTERNAL
+* Message *properties*:
+  * (required) `to`: either `event` or `e`
+  * (optional) `content-type`: The type of payload contained in the message body. The given content type will be used
+    in the AMQP message being forwarded downstream if not empty. Otherwise, the content type of the downstream
+    message will be set to `application/octet-stream` if the payload is not empty and no default content type has been
+    defined for the origin device or its tenant (see [Downstream Meta Data]({{< relref "#downstream-meta-data" >}}).
 * Message Body:
-  * (optional) Arbitrary payload
-* Message properties:
-  * (optional) Arbitrary properties (content-type, correlation-id, ...)
-* Disposition Frames:
-  * Accepted: Message successfully processed by the adapter.
-  * Released: Message cannot be processed and should be redelivered.
-  * Rejected: Adapter rejects the message due to:
-        * (`hono:bad-request`): A bad client request (e.g invalid content-type).
-        * (`amqp:unauthorized-access`): The adapter is disabled for tenant.
-        * (`amqp:precondition-failed`): Request not fulfilling certain requirements.
-        * (`amqp:resource-limit-exceeded`): Request rejected because the message limit for the given tenant is exceeded.
-
-This is the preferred way for devices to publish events. It is available only if the protocol adapter has been configured to require devices to authenticate (which is the default).
+  * (optional) Arbitrary payload contained in either a single AMQP *Data* or *AmqpValue* section.
+* Outcomes:
+  * `accepted`: The message has been successfully forwarded downstream.
+  * `released`: The message could not be processed by the adapter due to a (temporary) problem at the infrastructure level.
+    A common reason for this is that there is no active downstream consumer for messages from the device.
```

Comment:
The "no active downstream consumer" scenario doesn't usually apply for event messages here.

Reply:
The adapter has no influence on how the AMQP 1.0 Messaging Network is configured/implemented, has it? The statement seems to be true in any case, even if it never occurs ;-)

Reply:
Well, ok.
On that note: I think, we haven't clearly stated the differences between event and telemetry messages or the motivation for having both concepts anywhere in the documentation, have we? I think it's at least missing from the Telemetry/Event API pages. I can create a PR...

Reply:
I guess this could be added as part of #1571